package.json

```diff
@@ -5,13 +5,14 @@
   "main": "index.js",
   "scripts": {
     "dev": "node server.js",
+    "dev:debug": "node --inspect server.js",
     "build": "next build",
     "test": "jest",
     "start": "NODE_ENV=production node server.js",
     "format-staged": "pretty-quick --staged --no-restage --bail",
-    "format-check": "prettier --check '{.,pages/**,src/**,static/**}/*.{js,jsx,md}'",
+    "format-check": "prettier --check '{.,pages/**,public/static/docs/**,src/**}/*.{js,md}'",
     "lint-check": "eslint src pages",
-    "format-all": "prettier --write '{.,pages/**,src/**,static/**}/*.{js,jsx,md}'",
+    "format-all": "prettier --write '{.,pages/**,public/static/docs/**,src/**}/*.{js,md}'",
     "format": "prettier --write"
```

Comment:
so, how does `yarn format` work? is it documented somewhere?

Reply:
It just formats all the relevant js and md files. It's mentioned in https://dvc.org/doc/user-guide/contributing/docs#doc-style-guidelines-java-script-and-markdown:
![image](https://user-images.githubusercontent.com/1477535/71568340-bf7dda80-2a83-11ea-89a2-63b19412cb7a.png)